packages/cli/src/modules/insights/__tests__/insights.service.test.ts

```diff
@@ -509,6 +524,21 @@ describe('workflowExecuteAfterHandler', () => {
 			const accumulatedValues = allCompacted.reduce((acc, event) => acc + event.value, 0);
 			expect(accumulatedValues).toBe(batchSize);
 		});
+
+		test('compaction is running on schedule', async () => {
+			// ARRANGE
+			const insightsService = Container.get(InsightsService);
+
+			// spy on the compactInsights method to check if it's called
+			insightsService.compactInsights = jest.fn();
+
+			// ACT
+			// advance by 1 hour and 1 minute
+			jest.advanceTimersByTime(1000 * 60 * 60);
```

Comment:
Doesn't it need to be `1000 * 60 * 61` to be 1 hour 1 minute?

Reply:
Since we're using jest mock, the interval callback is ran immediately when reaching the interval, so no need to wait 1 second more